airbyte-integrations/connectors/destination-snowflake-cortex/destination_snowflake_cortex/indexer.py

```diff
@@ -0,0 +1,178 @@
+#
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+#
+
+import uuid
+from typing import Any, Iterable, Optional
+
+import dpath.util
+from airbyte._processors.sql.snowflake import SnowflakeSqlProcessor
+from airbyte._processors.sql.snowflakecortex import SnowflakeCortexSqlProcessor
+from airbyte.caches import SnowflakeCache
+from airbyte.strategies import WriteStrategy
+from airbyte_cdk.destinations.vector_db_based.document_processor import METADATA_RECORD_ID_FIELD, METADATA_STREAM_FIELD
+from airbyte_cdk.destinations.vector_db_based.indexer import Indexer
+from airbyte_cdk.models import (
+    AirbyteMessage,
+    AirbyteStateMessage,
+    AirbyteStateType,
+    AirbyteStreamState,
+    ConfiguredAirbyteCatalog,
+    DestinationSyncMode,
+    StreamDescriptor,
+    Type,
+)
+from destination_snowflake_cortex.config import SnowflakeCortexIndexingModel
+
+# extra columns to be added to the Airbyte message
+DOCUMENT_ID_COLUMN = "document_id"
+CHUNK_ID_COLUMN = "chunk_id"
+METADATA_COLUMN = "metadata"
+PAGE_CONTENT_COLUMN = "page_content"
```

Comment:
Minor nit. I'm not a huge fan of "page content" as a column name, just because it seems to somewhat bias towards web paradigms.

Wdyt of something like `document_content`, `document_contents`, `text_content`, or `contents`?

Reply:
changed it to document_content!